### DISCUSSION TITLE
请问下,RocketMQ有没有容灾和双向同步的方案

### DISCUSSION THREAD

**Discussion by wumingzhidewu** [apache/rocketmq#7954] [Q&A]
像Kafka有MirrorMaker, Pulsar有Geo Replicated,都能够实现异地多活架构中的双向同步,RocketMQ有没有相关的工具

**Comment by LittleBoy18** [comment]
社区里有一个rocketmq-replicator的组件
https://github.com/apache/rocketmq-externals/tree/rocketmq-replicator
